apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html

```diff
@@ -2147,8 +2147,8 @@
 
                                                 <label class="card-label text-right">Amount</label>
                                                 <div style="flex: 1;">
-                                                    <input type="tel" (blur)="calculateWhenTrxAltered(entry, transaction, true)"
-                                                        (input)="checkVoucherEntries()"
+
+                                                    <input type="tel" (blur)="calculateWhenTrxAltered(entry, transaction, true, $event)"
```

Comment:
`(input)="checkVoucherEntries()"` got removed please add again. Please check TEST (`giddh-2.0`) code as well as this is a breaking change